### DISCUSSION TITLE
Disable force skin saving in skin editor

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#19401] [Q&A]
Every change leads to skin saving after exit but it shouldnt (why there is a save button thought). Instead of saving there should be a  dialog window like "Do you want to save your changes?"

https://user-images.githubusercontent.com/86934170/180966873-c40cb66f-4ac8-4e5b-915f-f764a69372d5.mp4

**Comment by nagi-desuuu** [comment]
The UX would be terrible, though...
Getting interrupted by a dialog option never feels pleasant, so I feel like it can be treated as some sort of Autosave feature.

**Reply by Loreos7** [reply]
Then just don't save any changes until a user click the save button himself. For example: I put "Big black box" at the middle of the song selection screen, then returned home, then opened the song selection menu again and nothing has changed (box is disappeared).
